<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="C02PV4YEG8WM.local" name="atlasGroup.atlasID.pages.AddSurvey_43" tests="5" failures="0" timestamp="11 Sep 2015 15:09:42 GMT" time="193.503" errors="1">
  <testcase name="testActivate_Deactivate_65" time="41.876" classname="atlasGroup.atlasID.pages.AddSurvey_43"/>
  <testcase name="testActivate_Deactivate_65" time="21.144" classname="atlasGroup.atlasID.pages.AddSurvey_43"/>
  <testcase name="testActivate_Deactivate_65" time="38.034" classname="atlasGroup.atlasID.pages.AddSurvey_43"/>
  <testcase name="testActivate_Deactivate_65" time="44.636" classname="atlasGroup.atlasID.pages.AddSurvey_43">
    <error type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at atlasGroup.atlasID.pages.AddSurvey_43.testActivate_Deactivate_65(AddSurvey_43.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:38)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:382)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testActivate_Deactivate_65 -->
  <testcase name="testActivate_Deactivate_65" time="47.813" classname="atlasGroup.atlasID.pages.AddSurvey_43"/>
</testsuite> <!-- atlasGroup.atlasID.pages.AddSurvey_43 -->
